src/layouts/AssistenzaPage.jsx

```diff
@@ -484,26 +482,12 @@ export const AssistenzaPage = () => {
   };
 
   const formHandler = (event) => {
-    const notificationElement = document.querySelector('.notification');
-    const titleElement = notificationElement.querySelector('h5');
-    const descriptionElement = notificationElement.querySelector('p');
-    notificationElement.classList.add('show');
-    notificationElement.classList.add('success');
-    titleElement.innerHTML = `${successLabels.icon} ${successLabels.title}`;
-    descriptionElement.innerHTML = successLabels.description;
-
     setTimeout(() => {
+      window.location.assign('../../richiesta-inviata');
```

Comment:
I'd use `navigate` helper from gatsby to do it. As it already done [here](https://github.com/teamdigitale/padigitale2026.gov.it-site/blob/main/src/pages/supporto/faq.jsx#L5)